hivemind/optim/optimizer.py

```diff
@@ -187,11 +188,13 @@ def __init__(
         client_mode: bool = None,
         auxiliary: bool = False,
         grad_compression: CompressionBase = NoCompression(),
+        grad_rank_averager: Optional[str] = None,
```

Comment:
```suggestion
        grad_averager: Optional[Type[GradientAverager]] = None,
```

Reply:
(instead of str AND instead of grad_averager_opts that would be passed via partial)